tensorflow/workspace.bzl

```diff
@@ -52,6 +52,30 @@ temp_workaround_http_archive = repository_rule(
       "strip_prefix": attr.string(default = ""),
    })
 
+def _repos_are_siblings():
+  return Label("@foo//bar").workspace_root.startswith("../")
+
+def _curl_repo_impl(ctx):
```

Comment:
Would it be possible to refactor this to use `temp_workaround_http_archive` in workspace.bzl?